packages/hash/frontend/src/components/RemoteBlock/RemoteBlock.tsx

```diff
@@ -57,12 +58,34 @@ export const RemoteBlock: FunctionComponent<RemoteBlockProps> = ({
     ...graphProperties,
   });
 
+  useHookEmbedderService(wrapperRef, {
+    callbacks: {
+      async hook({ data }) {
+        if (data?.type === "text" && data.path === "$.text") {
+          editableRef?.(data.node);
+
+          const hookId = data.hookId ?? uuid();
+          return { data: { hookId } };
+        }
+
+        return {
+          errors: [{ code: "NOT_IMPLEMENTED", message: "Improper hook" }],
+        };
+      },
+    },
+  });
+
   useEffect(() => {
     if (graphService) {
       graphService.blockEntity({ data: graphProperties.blockEntity });
     }
   }, [graphProperties.blockEntity, graphService]);
 
+  useEffect(() => {
+    onBlockLoaded?.();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Do we need this? 

On the current implementation, it looks like if `onBlockLoaded` overfires it doesn't have any practical impact. If it changes in future so that it does, it might also change to need to be in the dependency array.

If we need this can we add to the comment line to explain why?

Reply:
I added this for testing something, but forgot to remove it. Again, my bad. I should have left this in draft and gone over it myself today. 

Removed in https://github.com/hashintel/hash/commit/7ea413260f1abd2b55cdb18559de572c598fb6bf